js/grid.custom.js

```diff
@@ -417,7 +417,7 @@ $.jgrid.extend({
 								break;
 							}
 						}
-						var st = soptions.searchtitle !== null ? soptions.searchtitle : p.operandTitle;
+						var st = soptions.searchtitle != null ? soptions.searchtitle : p.operandTitle;
```

Comment:
Just a quick question: Wouldn't it be better to use

```javascript
soptions.searchtitle !== undefined
```

`!=` will trigger a jshint warning (if we ever use the settings we talked about last week)